EligibilityCheckActivity/__tests__/handler.test.ts

```diff
@@ -33,17 +33,40 @@ const aINPSDsu: ConsultazioneSogliaIndicatoreResponse = {
   }
 };
 
-const mockConsultazioneSogliaIndicatore = jest.fn(() =>
-  fromEither(right(aINPSDsu))
-);
+const anotherFiscalCode = "CCCDDD80A01C123D" as FiscalCode;
+
+const anInvalidINPSDsu: ConsultazioneSogliaIndicatoreResponse = {
+  Esito: EsitoEnum.OK,
+  IdRichiesta: 1,
+
+  DatiIndicatore: {
+    DataPresentazioneDSU: new Date(),
+    PresenzaDifformita: SiNoTypeEnum.NO,
+    ProtocolloDSU: "PROTOCOLLO-DSU",
+    SottoSoglia: SiNoTypeEnum.SI,
+    TipoIndicatore: "ISEE Semplice",
+
+    Componenti: [
+      { CodiceFiscale: anotherFiscalCode, Cognome: "AAA", Nome: "BBB" }
+    ]
+  }
+};
+
```

Comment:
We can use `__mocks__/mocks.ts`, we have a collection of useful value objects that can be used consistently in every test

Reply:
This _invalid_ INPS DSU is invalid only inside this test because the requester fiscal code isn't inside the DSU itself.

Reply:
I see, it's just that it could be used by other suites to test _what if I do X passing an invalid DSU?_. Anyway it's totally fine to define it here. 

PS: https://github.com/pagopa/io-functions-bonus/blob/master/__mocks__/mocks.ts#L42

Reply:
this code is removed in 233cd6a